src/asciidoc/docs/blueprints/customVCC.adoc

```diff
@@ -0,0 +1,208 @@
+= Cat Selector VCC
```

Comment:
```suggestion
= Cat selector VCC
```
I was really inconsistent in how I converted these page titles, but I'd like to standardize on sentence case. I have some cleanup to do on the rest of the site :)